lib/quest/quest_watcher.rb

```diff
@@ -115,7 +109,7 @@ def start_watcher
       Quest::LOGGER.info("Initializing watcher watching for changes in #{quest_watch}")
       @watcher = FileWatcher.new(quest_watch)
       @watcher_thread = Thread.new(@watcher) do |watcher|
-        watcher.watch do |f|
+        watcher.watch do |changed_file_path|
           run_specs
```

Comment:
I'm not actually doing anything with that variable. When the watcher is triggered, it runs the contents of the watch block with the path of the changed variable bound to that variable. I left it in as a placeholder. I'll put in an INFO level debug message that uses that so it's a little more clear what the variable is doing and so we can see what change is triggering the run.